module/Database/view/database/member/subscribe.phtml

```diff
@@ -523,131 +387,3 @@ verwerken van mijn persoonsgegevens volgens haar privacybeleid.') ?>
         validator = new InputValidator(document.getElementById("<?= $form->getAttribute('id'); ?>"));
     });
 </script>
-
-<?php else: ?>
-    <h1 class="hidden-print"><?= $this->translate('Gefeliciteerd! Je bent geregistreed als lid van Studievereniging GEWIS') ?>
-        <br>
-        <small><?= $this->translate('Na betaling van je lidmaatschapsgeld ben je GEWIS-lid!') ?></small></h1>
-<!--Use the code below to add some nice pop-up or promotion to the subscription email.-->
-<!--    <div class="alert alert-info">-->
-<!--        Liked the introduction week? GEWIS also organizes a follow-up day after the introduction week: the FLUP!-->
-<!--        During the FLUP you get to know even more fellow students and GEWIS as an association through fun activities and a BBQ.-->
-<!--        This event will take place on Saturday, September 12. Tickets can be bought on <a href="https://gewis.nl/FLUP">https://gewis.nl/FLUP</a> for &euro;3,-.-->
-<!--    </div>-->
```

Comment:
Because I have removed this output and not added it to the `checkout/complete` page (as you can technically go back to that page to see the information again due to the `stripe_session_id`) and also made it into an e-mail using the new template, any messages that do special promotions (e.g., for FLUP) should be added to that e-mail instead.